### PR TITLE
add r-rockchalk

### PR DESCRIPTION
**Pull request by izahn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by izahn** [comment]
Needs
 r-kutils

**Comment by izahn** [comment]
@conda-forge admin please restart ci

**Comment by izahn** [comment]
@conda-forge-admin please restart ci

File: recipes/r-rockchalk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    functions are used to produce the statistics lectures in <http://pj.freefaculty.org/guides>.
+    Includes regression diagnostics, regression tables, and plots of interactions and
+    "moderator" variables. The emphasis is on "mean-centered" and "residual-centered"
+    predictors. The vignette 'rockchalk' offers a fairly comprehensive overview.  The
+    vignette 'Rstyle' has advice about coding in R.  The package title 'rockchalk' refers
+    to our school motto, 'Rock Chalk Jayhawk, Go K.U.'.
+
+  license_family: GPL3
+  license_file:
+    - LICENSE.txt
```
Comment (cbrueffer) [inline recipes/r-rockchalk/meta.yaml:62]:
```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

File: recipes/r-rockchalk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  run:
+    - r-base
+    - r-mass
+    - r-cardata
+    - r-kutils
+    - r-lme4
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
```
Comment (cbrueffer) [inline recipes/r-rockchalk/meta.yaml:45]: The comment serves no purpose and can be removed.

File: recipes/r-rockchalk/meta.yaml (hunk 1 above)
Comment (cbrueffer) [reply recipes/r-rockchalk/meta.yaml:62]: Please remove the `license.txt` file and instead reference the license file that `r-base` already installs.

**Comment by cbrueffer** [comment]
The `LICENSE.txt` file can be removed as well.

**Pull request merged**